Sources/MarigoldSwift/EngageBySailthru+Concurrency.swift

```diff
@@ -185,5 +186,20 @@ extension EngageBySailthru {
             logAbandonedCart(abandonedCart, withResponse: ClosureBuilder.voidErrorClosure(continuation))
         })
     }
+    
+    /**
```

Comment:
nitpick - we should maybe move this up to where the other attributes methods are so they're grouped together for convenience